fpx/src/commands/client/requests.rs

```diff
@@ -42,3 +50,82 @@ async fn get_request(args: GetArgs) -> Result<()> {
 
     Ok(())
 }
+
+/// Temporary inline this code into this module
```

Comment:
This comment can be removed (and others like it). Initially I was under the impression that we should create separate files for each get/post/invoke. But I think it is fine to have 1 file for each of the commands per resource.